app/client/sagas/authSagas.ts

```diff
@@ -177,6 +178,13 @@ function* saveOrgId(
 function* watchSaveOrgId() {
   yield takeEvery(LoginActionTypes.UPDATE_ACTIVE_ORG, saveOrgId);
 }
+
+export function* apiLogout() {
+  yield call(logoutAPI);
+  yield call(removeSessionToken);
+  yield call(removeOrgIds);
```

Comment:
Can you explain the reasoning for why there's a separate `apiLogout` and `logout` function?

Reply:
Two reasons! One is that we don't necessarily want to blacklist every time we flush the session token (namely when we're refreshing the token, we don't want to invalidate the token we're about to refresh with).   
The other is that we sometimes call `logout` with no token at all, or with an expired token (Like when you go directly to the `/login` page). These cases would just fail and there's no use wasting an API call on them IMO

Reply:
That makes sense